modules/cactus-web/src/Header/Header.tsx

```diff
@@ -34,26 +34,85 @@ export const Header: HeaderType = ({ children, bgColor = 'lightContrast', ...res
 
   return (
     <StyledHeader bgColor={bgColor} {...rest}>
-      <HeaderColumn mainColumn>
+      <MainColumn>
         {childrens.find(
           (child) =>
             (child as ChildElement).type.displayName === 'HeaderBreadcrumbRow' && <>{child}</>
         )}
         {childrens.find(
           (child) => (child as ChildElement).type.displayName === 'HeaderTitle' && <>{child}</>
         )}
-      </HeaderColumn>
+      </MainColumn>
       {childrens.some((i) => (i as ChildElement).type.displayName === 'HeaderItem') && (
-        <HeaderColumn>
+        <ItemsColumn>
           {childrens.filter(
             (child) => (child as ChildElement).type.displayName === 'HeaderItem' && <>{child}</>
           )}
-        </HeaderColumn>
+        </ItemsColumn>
       )}
     </StyledHeader>
   )
 }
 
+const columnStyles = css`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+`
+
+const MainColumn = styled.div`
+  ${columnStyles}
+  ${(p) => `
+    ${p.theme.mediaQueries?.small}{
+      padding-top: 0px;
+      min-width: 40%;
+      > div:not(:first-child) {
+        margin-left: 0px;
+      }
+      align-items: flex-start;
+    }
+    flex: 1 1;
+    > h2 {
+      max-width: 100%;
+      flex-shrink: 0;
+    }
+  `}
+`
+
+const ItemsColumn = styled.div`
+  ${columnStyles}
+  padding-top: 24px;
+  > div:not(:first-child) {
+    margin-top: 8px;
+  }
+
+  ${(p) => `
+    ${p.theme.mediaQueries?.small}{
+      align-items: flex-end;
+      flex: 1 1 0px;
+      hyphens: auto;
+      overflow-wrap: break-word;
+      padding-top: 0px;
+      max-width: 100%;
+      word-wrap: break-word;
+      > div {
+        margin-top: 0px;
+        margin-left: 8px;
+        word-wrap: break-word;
+        max-width: 100%;
+        hyphens: auto;
+      }
+    }
+
+    ${p.theme.mediaQueries?.medium}{
+      flex-direction: row;
+      align-items: center;
+      flex: 0 1 auto; 
```

Comment:
```suggestion
      flex: 0 1 auto; 
      > div:not(:first-child) {
        margin-top: 0px;
      }
```

Other than that I'm ready to approve this